swift/vapor-framework/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "server",
+    platforms: [
+        .macOS("10.15")
```

Comment:
does it force compilation to be on `OS X` ? for the first **production** version, we will use `droplets` (from https://digitalocean.com) so only `linux`

PS : Using specialized cloud / infrastructure could be a great feature, but far far away in the roadmap :stuck_out_tongue:

Reply:
No, this is simply required because if you are on macOS, you need to be on version 10.15 or later. This will still work fine on Linux.

Reply:
Thanks for clarification :ok_hand: